config/alerting.go

```diff
@@ -0,0 +1,67 @@
+// promconfig
+//Copyright 2020 Percona LLC
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+// Original repository: https://github.com/prometheus/prometheus
```

Comment:
I specified the repository instead of a concrete file because they don't match one-to-one, is it ok?